src/server/db_slice.cc

```diff
@@ -218,6 +218,38 @@ unsigned PrimeEvictionPolicy::Evict(const PrimeTable::HotspotBuckets& eb, PrimeT
   return 1;
 }
 
+constexpr uint32_t kClearStepSize = 1024;
```

Comment:
Maybe write all this functionality using class name AsyncDeletetion

Reply:
this class won't have state because the only state we have is `__thread ClearNode* clear_head = nullptr;`

I will group the functionality into static methods.